### DISCUSSION TITLE
Standalone panel plugins

### DISCUSSION THREAD

**Discussion by elviosak** [lxqt/lxqt#2033] [General]
i setup a repo to compile a plugin separate from the panel, makes it easier for testing some stuff here, if someone else is interested it's here: https://github.com/slidinghotdog/plugin-example 
it should also work just copying to panel folder and adjusting cmake stuff.

also are there any plans for lxqt to have some panel plugins in separate repos?

**Comment by tsujan** [comment]
> also are there any plans for lxqt to have some panel plugins in separate repos?

Haven't heard of any but, IMO, it's good to talk about it here.

Most LXQt devs that have made lxqt-panel what it is haven't been active for a while; I hope they'll return soon and discuss it.
